submissions/jeff-james/scripts/article.js

```diff
@@ -73,18 +93,31 @@ Article.fetchAll = function() {
       3. And then render the index page (What method was that?) */
     $.getJSON('data/hackerIpsum.json', function(data){
       console.log(data);
-      // var jsonData = 
       Article.loadAll(data);
       localStorage.setItem('data', JSON.stringify(Article.all));
       articleView.initIndexPage();
+    }).success(function(data, message, xhr) {
```

Comment:
Also here. You don't need the success function, you can just pass data, msg, xhr to the $.getJSON callback function and get the eTag that way.